docs/DeveloperGuide.md

```diff
@@ -9,7 +9,7 @@ title: Developer Guide
 
 ## **Acknowledgements**
 
-* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
+* {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
```

Comment:
Please revert it back to the addressbook3 reference. That’s supposed to be there.

Reply:
noted with thanks!